report/arcticreport.Rmd

```diff
@@ -188,8 +153,8 @@ Ask Robyn or the lead MetacatUI dev to give you access to Google Analytics if yo
 # Plots
 
 ```{r}
-qs <- "2023-05-01"
-qe <- "2023-07-31"
+qs <- "2023-08-01"
+qe <- "2023-10-31"
```

Comment:
can we change this so it isn't hard coded?

Reply:
I pushed a new commit, where I added one line to change the current quarter to remove the hard coded dates